docs/spec.json

```diff
@@ -0,0 +1,383 @@
+{
+  "name": "corectl",
+  "info": {
+    "title": "Specification for corectl",
+    "description": "corectl contains various commands to interact with the Qlik Associative Engine. See respective command for more information",
+    "version": "v0.4.0",
```

Comment:
Version should be a string according to semantic versioning which does not allow 'v' infront of the X.Y.Z

Reply:
fixed https://598-146282949-gh.circle-artifacts.com/0/home/circleci/corectl/docs/spec.json